### PR TITLE
CLDR-11585 move cldr-apps to maven

### PR DESCRIPTION
**Pull request by srl295**
[CLDR-11585]

- use 'hack-install-jars.sh' to install jars locally
- pom is somewhat fragile, has to keep in sync with cldr.jar

fyi @hagbard @btangmu 

##### Checklist

- [x] Issue filed: https://unicode-org.atlassian.net/browse/CLDR-_____
- [x] Updated PR title and link in previous line to include Issue number

[CLDR-11585]: https://unicode-org.atlassian.net/browse/CLDR-11585

### CODE REVIEW

**Comment by srl295** [comment]
question about when is a 'good time' to go in, could be just past GA

this updated cldr-apps (ST) leaves the rest alone.  Impact on @btangmu and anyone else's branches?

no feedback so far

**Comment by btangmu** [comment]
FYI, I have work-in-progress changes (for [CLDR-11877](url) and comments for [CLDR-12020](url)), in local branches, to a few of the files that this PR will move (without otherwise modifying), in directories including tools/cldr-apps/src/org/unicode/cldr/web/. It wouldn't be very hard for me to redo my changes in the new locations. Still, it might add a little extra time and confusion. I don't want to block this PR if it's ready to go. Alternatively I could prepare my own PR by Monday or Tuesday to be merged first, unless that would make things harder for Steven.

**Comment by srl295** [comment]
@btangmu , lets wait until cldr release.  Let's coordinate and get your branch changes in. There's no rush on this. It's much easier for me to rebase than the other way around. We can discuss Wednesday.

Obtener Outlook para Android<https://aka.ms/ghei36>

________________________________

**Comment by macchiati** [comment]
@btangmu , Steven; now that the release is done, is it time to push this forward?

File: tools/cldr-apps/pom.xml [hunk 1]
```diff
@@ -0,0 +1,239 @@
...
+      <version>10.10.1.1</version>
+      <!-- <scope>test</scope> -->
+    </dependency>
+
+
+<!-- https://mvnrepository.com/artifact/com.google.guava/guava -->
+<dependency>
+    <groupId>com.google.guava</groupId>
+    <artifactId>guava</artifactId>
+    <version>28.1</version>
```
Comment (srl295) [inline tools/cldr-apps/pom.xml]: ^ when we want to bump the guava version, just change the number here…

File: tools/cldr-apps/readme.txt [hunk 2]
```diff
@@ -1,107 +1,39 @@
...
+Copyright 1991-2019 Unicode, Inc.
+All Rights Reserved. Terms of use: http://www.unicode.org/copyright.html
 ------------------------------------------------------------
 
 Note: this document is somewhat out of date. see 
 	http://cldr.unicode.org/development/running-survey-tool
+for the latest instructions, other than maven instructions below:
 
-for the latest instructions.
+** BUILDING WITH MAVEN
```
Comment (srl295) [inline tools/cldr-apps/readme.txt]: ^ updated the readme a bit.

**Comment by btangmu** [comment]
continuous-integration/travis-ci/pr — The Travis CI build failed: "Could not find artifact com.google.guava:guava:jar:28.1 in google-maven-central (https://maven-central.storage-download.googleapis.com/repos/central/data/)"?

**Comment by srl295** [comment]
can someone make sure they can actually build this before i merge?  The links next to the 'squash and merge' button will let you checkout a separate working copy, if you like:

>  You can also open this in GitHub Desktop or view command line instructions.

**Comment by srl295** [comment]
> I get 4 errors in Eclipse:
> 
> 1. Missing artifact com.ibm.icu:icu-utilities:jar:0.1-SNAPSHOT	pom.xml	/cldr-apps	line 64	Maven Dependency Problem
> 2. Missing artifact org.unicode.cldr:cldr-api:jar:0.1-SNAPSHOT	pom.xml	/cldr-apps	line 58	Maven Dependency Problem
> 3. The container 'Maven Dependencies' references non existing library '/Users/tbishop/.m2/repository/org/unicode/cldr/cldr-api/0.1-SNAPSHOT/cldr-api-0.1-SNAPSHOT.jar'	cldr-apps		Build path	Build Path Problem
> 4. The project cannot be built until build path errors are resolved	cldr-apps		Unknown	Java Problem

please make sure to:
1. build cldr's jars ( `ant jar` in tools/cldr )
2. run `install-cldr-jars.sh`

**Comment by btangmu** [comment]
...
> please make sure to:
> 
>     1. build cldr's jars ( `ant jar` in tools/cldr )
> 
>     2. run `install-cldr-jars.sh`

Still installing Maven... Sorry my comment was premature. I'll keep you posted.

**Comment by btangmu** [comment]
I think the plan is to just move files first, and save the actual Mavenization for a subsequent PR?

**Comment by srl295** [comment]
Just a test:

CLDR-11585

**Pull request closed without merging**